### ISSUE TITLE
error: 'add_2_calendar/add_2_calendar-Swift.h' file not found

### ISSUE THREAD

**Issue by kushwahashiv** [ja2375/add_2_calendar#4]
Xcode build done.                                            2.1s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **

Xcode's output:
↳
    === BUILD TARGET path_provider OF PROJECT Pods WITH CONFIGURATION Debug ===
    /Users/shiv/Applications/flutter/.pub-cache/hosted/pub.dartlang.org/add_2_calendar-1.0.2/ios/Classes/Add2CalendarPlugin.m:2:9: fatal error: 'add_2_calendar/add_2_calendar-Swift.h' file not found
    #import <add_2_calendar/add_2_calendar-Swift.h>
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    1 error generated.

Could not build the application for the simulator.
Error launching application on iPhone XR.

**Comment by bkbonner** [comment]
Yeah, I'm seeing the same thing.  :(

I updated cocoapods to 1.5.3.1
`brew upgrade cocoapods`

From the ios directory, I ran:
```
pod setup
pod install
```

result:

```
Launching lib/main.dart on iPhone 7 in debug mode...
Xcode build done.                                            1.8s
Failed to build iOS app
Error output from Xcode build:
↳
** BUILD FAILED **
Xcode's output:
↳
=== BUILD TARGET shared_preferences OF PROJECT Pods WITH CONFIGURATION Debug ===
/Users/bbonner/development/flutter/.pub-cache/hosted/pub.dartlang.org/add_2_calendar-1.0.2/ios/Classes/Add2CalendarPlugin.m:2:9: fatal error: 'add_2_calendar/add_2_calendar-Swift.h' file not found
#import <add_2_calendar/add_2_calendar-Swift.h>
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
Could not build the application for the simulator.
```

**Comment by bkbonner** [comment]
I think this is just the result of an extraneous import....it looks like the file might have been renamed from 
add_2_calendar/add_2_calendar-Swift.h
to
SwiftAdd2CalendarPlugin.swift

**Comment by bkbonner** [comment]
In:

Add2CalendarPlugin.m

it shows:

```
#import "Add2CalendarPlugin.h"
#import <add_2_calendar/add_2_calendar-Swift.h>

@implementation Add2CalendarPlugin
+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
  [SwiftAdd2CalendarPlugin registerWithRegistrar:registrar];
}
@end
```

The error message I was reporting seems like it's a bad import --- I can't find add_2_calendar-Swift.h anywhere in the file.

Is the add_2_calendar package is numbered 1.0.2 referring to the release tagged 1.0.2?   Is this referring to the same code?


I was trying to run this project:  https://github.com/jesusrp98/spacex-go  when I ran into this issue.

Thank you.

**Comment by ja2375** [comment]
It's not a bad import, it is a bridging header.

I tried the project and it threw that error at first.
It seems that the project has not been created with Swift support, which is really weird. I tend to think that some project file didn't sync to github, as i suppose the author of that project was able to run it.

I was able to fix that just by creating an empty project **with Swift support**, and just copying all the project files (code, assets & pubspec) and running it on the simulator.

Hope that helps :)

**Comment by bkbonner** [comment]
ah...you're suggesting to create a new flutter project....and copy the dart related content over....and then run it that way.   that seems reasonable.

At that point I can compare the two projects and see what the differences are (what gets added for iOS Swift?

The add_2_calendar-Swift.h file seemed weird to me....if that file doesn't exist anywhere in the project.

**Comment by ja2375** [comment]
Considering this solution worked for you as no reply.

**Comment by bkbonner** [comment]
sorry for the delayed response.   your solution was complete and I appreciate the link.   I had spoken w/ someone I thought was familiar with iOS, but your answer makes sense. 

I am testing it now.

**Comment by ja2375** [comment]
Don't worry :)

**Comment by bkbonner** [comment]
I was able to get it to work by copying over the lib and assets directory from the old project to a flutter project created with -i swift. 

I ran diff -rq spacex-go spacex-swift

where space-go is the orig project and spacex-swift is project created with -i swift  and I've attached a list of files that were different.  Too many to mention.

There were so many differences, it was easier to just move the lib and assets directory from the spacex-go directory to spacex-swift and it seemed to work ok.

[differences.txt](https://github.com/ja2375/add_2_calendar/files/2815506/differences.txt)
